src/lib/Scenes/SaleInfo/SaleInfo.tsx

```diff
@@ -123,15 +121,82 @@ export const SaleInfo: React.FC<Props> = ({ sale, me }) => {
           {renderLiveBiddingOpening()}
         </Flex>
 
-        {/*  Live Auction Notice */}
         {Boolean(sale.liveStartAt) && <AuctionIsLive />}
-        {/*  Auction Support */}
+        {!!sale.isWithBuyersPremium && <BuyersPremium sale={sale} />}
         <AuctionSupport />
       </Join>
     </ScrollView>
   )
 }
 
+const createPremiumDisplay = (props: { sale: SaleInfo_sale }) => {
+  return (
+    <FlatList
+      data={props.sale.buyersPremium}
+      renderItem={({ item, index }) => {
+        return <BuyersPremiumItem sale={props.sale} currentValue={item} index={index} />
+      }}
+      keyExtractor={(item, _index) => item?.amount || "0"}
+    />
```

Comment:
(nonblocking) It's a little odd to use a `FlatList` for this since it creates a new scrolling context in the middle of an existing scrolling context. It'd be more idiomatic to just map over the buyers' premium items.

```suggestion
    props.sale.buyersPremium.map(item =>
      <BuyersPremiumItem sale={props.sale} currentValue={item} index={index} key={index}/>
    )
```